test/integration/029_docs_generate_tests/ref_models/schema.yml

```diff
@@ -29,3 +29,16 @@ sources:
         columns:
           - name: id
             description: "{{ doc('column_info') }}"
+
+exposures:
+  - name: notebook_exposure
+    type: notebook
+    depends_on:
+      - ref('model')
+      - ref('second_model')
```

Comment:
This test "project" doesn't have models named `model` or `second_model`, which is causing the current error. You could swap these out for any of:
```
- ref('ephemeral_summary')
- ref('view_summary')|
- source('my_source', 'my_table')
```
And then swap out the values of `depends_on.nodes` in `gest_docs_generate.py` down below.